projects/go-lib/src/lib/components/go-timepicker/go-timepicker.component.html

```diff
@@ -0,0 +1,51 @@
+<label
+  [attr.for]="id"
+  class="go-form__label"
+  [ngClass]="{ 'go-form__label--dark': theme == 'dark' }"
+>
+  {{ label }}
+  <go-required-text [control]="control"></go-required-text>
+</label>
+<div class="go-timepicker">
+  <input
```

Comment:
Should we change the cursor on hover for this input since we're opening the time picker instead of actually allowing a user to change the input?
```
cursor: pointer;
```